pyccel/stdlib/ndarrays/ndarrays.h

```diff
@@ -32,17 +32,21 @@ typedef struct  s_slice
     int32_t step;
 }               t_slice;
 
+/*
+** Map e_types enum to numpy NPY_TYPES enum
+** ref: numpy_repo: numpy/numpy/core/include/numpy/ndarraytypes.h
+*/
 enum e_types
 {
         nd_bool,
         nd_int8,

```

Comment:
Maybe it would be better to provide numbers for everything in this enum to be consistent